lib/runner.js

```diff
@@ -46,6 +50,12 @@ util.inherits(Runner, EventEmitter);
 module.exports = Runner;
 
 optionChain(chainableMethods, function (opts, title, fn) {
+	if (opts.todo && typeof title === 'function') {
+		throw new TypeError('Expected a string');
```

Comment:
The error message could be more informative. Perhaps `TODO tests need a title` or something along those lines.